app/controllers/projects_controller.rb

```diff
@@ -89,7 +89,73 @@ def index # rubocop:disable Metrics/AbcSize
   end
 
   def new
-    render layout: "no_menu"
+    @project = if params[:template_id].present?
+                 @template = Project.find_by(id: params[:template_id])
```

Comment:
```suggestion
                 @template = Project.visible.find_by(id: params[:template_id])
```

As we are copying data from that project, not using the visible scope here might leak data from the existing project that a user does not have access to. And we should also probably only allow this for templated projects, shouldn't we? But I am not 100% sure about this

```suggestion
                 @template = Project.visible.find_by(id: params[:template_id], templated: true)
```